homeassistant/config_entries.py

```diff
@@ -331,6 +338,10 @@ async def async_setup(
 
         if integration is None:
             integration = await loader.async_get_integration(hass, self.domain)
+            self._integration_for_domain = integration
+
+        if tries is not None:
```

Comment:
We should be able to drop this block and the kwargs as we don’t have to pass this around anymore